src/ng/messages/user_state.rs

```diff
@@ -0,0 +1,90 @@
+use crate::ng::{FromIrcMessage, InvalidMessage, Validator};
+use crate::ng::{IrcMessage, Str, StrIndex, TagIndices, Tags};
+
+use crate::{color::Color, parse_badges, parse_emotes, Badge, BadgeInfo, Emotes};
+
+#[derive(Debug, Clone, PartialEq)]
+pub struct UserState<'t> {
+    raw: Str<'t>,
+    tags: TagIndices,
+    channel: StrIndex,
+}
+
+impl<'t> UserState<'t> {
```

Comment:
missing user-id tag getter

Reply:
I also don't see that in https://dev.twitch.tv/docs/irc/tags#userstate-twitch-tags

Do you have a sample for a test?